Source/Base/Base/Math/Interpolation.h

```diff
@@ -0,0 +1,403 @@
+#include "Math.h"
+
+namespace Spline
+{
+    class Interpolation
+    {
+    public:
+        static void ClampAnim(f32& t)
```

Comment:
I'd return the f32 instead of directly modifying it. (This goes for any similar function like "ClampPolynomialDegree" that takes in an integer or float for the entire purpose of mathematical operations like clamp, min, max etc.)